packages/story-editor/src/components/reorderable/useReordering.js

```diff
@@ -36,9 +36,10 @@ function useReordering(onPositionChange, numChildren) {
   const handleStartReordering = useCallback(
     ({ position: currentPos, onStartReordering = () => {} }) =>
       (evt) => {
-        // Only allow reordering with non-modified click on non-background element.
```

Comment:
The "non-background element" part was left from before the _reorderable_ was generalized out of the layer panel, so this comment does not belong here anymore.